doc/sphinx/source/recipes/recipe_gier20bg.rst

```diff
@@ -0,0 +1,231 @@
+.. _recipes_gier20bg:
+
+Spatially resolved evaluation of ESMs with satellite column-averaged CO\ :sub:`2`
+=================================================================================
+
+Overview
+--------
+
+This recipe reproduces the figures of Gier et al. (2020). It uses satellite
+column-averaged CO\ :sub:`2` data to evaluate ESMs by plotting several
+quantities such as timeseries, seasonal cycle and growth rate in different
+areas.
+
+Available recipes and diagnostics
+---------------------------------
+
+Recipes are stored in esmvaltool/recipes/
+
+    * recipe_gier20bg.yml
+
+Diagnostics are stored in diag_scripts/
+
+Diagnostics are stored in esmvaltool/diag_scripts/xco2_analysis/
+
+    * carbon_plots.ncl: plot script for panel plots
+    * delta_T.ncl: IAV of growth rate against growing season temperature - Figure C1
+    * global_maps.ncl: global maps for seasonal cycle amplitude - Figures 5, 6
+    * main.ncl: Timeseries and histogram - Figures 3, 4
+    * panel_plots.ncl: scatter plot of SCA/GR vs variable - Figures 7, 9, B1, B2
+    * sat_masks.ncl: data coverage of input data - Figures 1, 8
+    * stat.ncl: auxiliary functions for GR, SCA computation
+    * station_comparison.ncl: - comparison of surface and column data - Figure 2
+
+
+User settings in recipe
+-----------------------
+
+#. Preprocessor
+
+    * ``conv_units``: converts units to plot-units
+    * ``mmm_ref``: calculates multi-model mean and regrids to ref dataset
+    * ``mmm_2x2``: computes multi-model mean on 2x2 grid
+    * ``mmm``: computes multi-model mean for 3D variable, 5x5 grid with specific
+      pressure levels
+
+#. Script xco2_analysis/delta_T.ncl
+
+    * Required diag_script_info attributes:
+        * ``region``: region to average over
+        * ``masking``: the kind of masking to apply prior to region average
+            * options: obs, land, sciamachy, gosat, none
+        * ``var_order``: First main variable, then temperature variable to compare
+
+    * Optional diag_script_info attributes:
+        * ``styleset``: styleset for color coding panels
+        * ``output_file_type``: output file type for plots, default: png
```

Comment:
Would it be possible to use this option from the config file? Applies to all relevant scripts.

Reply:
It was originally from the config file, but then there was the discussion about deprecating these functions in the config, and it seems that it's still a bit of an ongoing conversation, so not sure what's the best option here. I could leave the option in but default to config file plottype?

Reply:
Yes, using the one from the config as default would be nice! I think we agreed on not deprecating it, so in principle you could remove it from the diagnostics.